tests/tests_fabric/loggers/test_csv.py

```diff
@@ -150,33 +153,52 @@ def test_append_columns(tmp_path):
 
     # initial metrics
     logger.log_metrics({"a": 1, "b": 2})
+    _assert_csv_content(
+        logger.experiment.metrics_file_path,
+        expected_headers={"step", "a", "b"},
+        expected_content=[{"step": "0", "a": "1", "b": "2"}],
+    )
 
     # new key appears
-    logger.log_metrics({"a": 1, "b": 2, "c": 3})
-    with open(logger.experiment.metrics_file_path) as file:
-        header = file.readline().strip()
-        assert set(header.split(",")) == {"step", "a", "b", "c"}
+    logger.log_metrics({"a": 11, "b": 22, "c": 33})
+    _assert_csv_content(
+        logger.experiment.metrics_file_path,
+        expected_headers={"step", "a", "b", "c"},
+        expected_content=[
+            {"step": "0", "a": "1", "b": "2", "c": ""},
+            {"step": "0", "a": "11", "b": "22", "c": "33"},
+        ],
+    )
 
     # key disappears
-    logger.log_metrics({"a": 1, "c": 3})
-    with open(logger.experiment.metrics_file_path) as file:
-        header = file.readline().strip()
-        assert set(header.split(",")) == {"step", "a", "b", "c"}
-
-
-def test_rewrite_with_new_header(tmp_path):
```

Comment:
Could you keep my previous test without rewriting it? I'd like to keep it the same way to avoid regressions.